src/solo12.cpp

```diff
@@ -163,7 +165,8 @@ void Solo12::acquire_sensors()
         slider_positions_(i) = double(slider_positions_vector_[i+1]) / 1024.;
     }
 
-    active_estop_ = slider_positions_vector_[0] == 0;
+    // Active the estop if button is pressed or the estop was active before.
+    active_estop_ = slider_positions_vector_[0] == 0 || active_estop_;
```

Comment:
would `active_estop_ |= slider_positions_vector_[0] == 0` this work?

Reply:
Good point, I changed the code.